tests/integration/helpers/wrap_kakarot.py

```diff
@@ -30,6 +30,7 @@ async def _wrapped(contract, *args, **kwargs):
                 call = kakarot.execute_at_address(
                     address=evm_contract_address,
                     value=0,
+                    gas_limit=1000000,
```

Comment:
could we instead do something like:

```
                if "value" in kwargs:
                    value = kwargs["value"]
                    del kwargs["value"]
                else:
                    value = 0
```

for `gas_limit` so that we're able to update it easily in tests? (to be put before the `if`)